docker/docker-compose.yml

```diff
@@ -37,7 +37,7 @@ services:
       - 11800:11800
       - 12800:12800
     volumes:
-      - ./config:/apache-skywalking-apm-incubating/config:ro
+      - ./config:/skywalking/config:ro
```

Comment:
should be `./config:/apache-skywalking-apm-incubating-bin/config:ro` according to [OAP Dockerfile](https://github.com/apache/incubator-skywalking/blob/master/docker/oap/Dockerfile)